### PR TITLE
Updated design of card buttons

### PR DESCRIPTION
**Pull request by HakimSkaik**
description & video left on project board issue

### CODE REVIEW

File: src/components/Button/Button.scss [hunk 1]
```diff
@@ -43,31 +41,24 @@
       background-position: 100% 100%;
       background-size: 100% 100%;
       color: var(--background);
-      border-color: transparent;
+      border: 1px solid #a6c50c;
```
Comment (danielle-dsouza) [inline src/components/Button/Button.scss:44]:
```suggestion
      border: 1px solid var(--primary-light);
```

File: src/components/Button/Button.scss [hunk 2]
```diff
@@ -43,31 +41,24 @@
...
 
       &:hover {
         background-position: 0% 100%;
         background-size: 0% 100%;
-        color: white;
+        color: black;
+        background-color: #a6c50c;
```
Comment (danielle-dsouza) [inline src/components/Button/Button.scss:50]:
```suggestion
        background-color: var(--primary-light);
```

File: src/components/Button/Button.scss [hunk 3]
```diff
@@ -43,31 +41,24 @@
...
-      box-shadow: 2px 1000px 1px rgb(97, 94, 95) inset;
+      border: solid 1px white;
       color: white;
 
       &:hover {
-        color: black;
-        box-shadow: none;
-        transition: color 0.3s, box-shadow 0.3s;
+        color: white;
+        background-color: #6336bb;
```
Comment (danielle-dsouza) [inline src/components/Button/Button.scss:60]:
```suggestion
        background-color: var(--primary-dark);
```

File: src/components/Button/Button.scss [hunk 4]
```diff
@@ -43,31 +41,24 @@
...
+        border-color: #6336bb;
```
Comment (danielle-dsouza) [inline src/components/Button/Button.scss:61]:
```suggestion
        border-color: var(--primary-dark);
```

**Review by danielle-dsouza — CHANGES_REQUESTED** [review]
Hey! Looks good! I just noticed that the text of `Join us` should be `var(--primary-light)` not black by default! 

In this PR:
Screen Shot 2022-10-01 at 8 39 10 AM

Desire:
Screen Shot 2022-10-01 at 8 40 46 AM

**Comment by HakimSkaik** [comment]
Updated!

**Review by danielle-dsouza — CHANGES_REQUESTED** [review]
Hmm the `Join us` is still black for me.

Screen Shot 2022-10-01 at 2 11 25 PM

**Comment by HakimSkaik** [comment]
I just followed  all of your suggested changes; if it didn’t work would you
explain what you mean by black so i could fix it.

**Comment by danielle-dsouza** [comment]
> I just followed all of your suggested changes; if it didn’t work would you explain what you mean by black so i could fix it.

The text is black. It's supposed to be `var(--primary-light)` like in the image below
Screen Shot 2022-10-01 at 2 18 36 PM

**Comment by HakimSkaik** [comment]
ohh so sorry for not catching that earlier, would you check again now? Thank you!

**Review by danielle-dsouza — APPROVED** [review]
Looks 💯!

**Pull request merged**